Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:6
```

Comment:
Do you mean to grab it dynamically, or to just specify what is currently in `.nvmrc` (ie, `FROM node:6.7`)?

I don't believe there is a way to grab it dynamically in the Dockerfile itself, though we could potentially make a container that has `nvm` and have it use the repo's `.nvmrc` to turn on whichever node version is specified.